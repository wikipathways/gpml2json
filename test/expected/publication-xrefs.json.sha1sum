960845aede673952b3f3e31e772f8435f629700e ?-
