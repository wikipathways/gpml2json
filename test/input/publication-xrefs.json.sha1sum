6a888e17461af8f0ba4409007fc8316896ca8835 ?-
