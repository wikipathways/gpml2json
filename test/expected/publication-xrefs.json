{"pathway":{"@context":["https://cdn.rawgit.com/wikipathways/WpVocabularies/7a46a05/contexts/pvjs.jsonld"],"backgroundColor":"white","borderWidth":0,"color":"black","contains":["id9ee2ea1d","id8ee9f3be","id788e2f55","ed0ee","e114e","c9fbc","ccc39"],"drawAs":"rect","gpmlElementName":"Pathway","height":378.76221498371376,"kaavioType":"Group","name":"test case","padding":5,"type":["Pathway"],"width":412.7991886409736,"x":0,"y":0,"zIndex":0,"fontSize":12,"fontWeight":"bold","textAlign":"left","verticalAlign":"top","dataSourceVersion":"20130815","standardName":"test case","displayName":"test case","textContent":"Name: test case","isSimilarTo":"http://wikipathways.org/index.php/Special:SearchPathways?query=test%20case&doSearch=1","id":"hash://sha256/91d56d8da3a612d05fa14fa073999b142d1ca05694e4880bc1345992c8b8a562"},"entityMap":{"e114e":{"id":"e114e","textContent":"Hmgcr","wpType":"GeneProduct","citations":["publicationXrefe0d","publicationXrefc0d","publicationXrefcf5","publicationXrefb4f"],"x":108.2084690553746,"y":174.03908794788273,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":32768,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"dbConventionalName":"Entrez Gene","dbId":"15357","type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"ed0ee":{"id":"ed0ee","citations":["publicationXrefb8e","publicationXrefc86","publicationXrefd54"],"cellularComponent":"Mitochondria","x":291.0309446254072,"y":180.28175895765472,"color":"#c0c0c0","fontSize":10,"height":54.2459283387622,"borderWidth":3,"rotation":0,"drawAs":"Mitochondria","verticalAlign":"middle","width":80.3045602605863,"zIndex":16384,"textAlign":"center","backgroundColor":"transparent","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Shape","SingleFreeNode","PhysicalEntity","CellularComponent","Mitochondria"],"kaavioType":"SingleFreeNode","gpmlElementName":"Shape"},"c9fbc":{"id":"c9fbc","textContent":"Metabolite","wpType":"Metabolite","x":155.4397394136808,"y":328.76221498371376,"color":"#0000ff","fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":32768,"textAlign":"center","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","Metabolite"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"ccc39":{"id":"ccc39","textContent":"Pathway","wpType":"Pathway","citations":["publicationXrefba5","publicationXrefe88"],"x":302.7991886409736,"y":298.316430020284,"color":"#14961e","fontSize":12,"fontWeight":"bold","height":20,"drawAs":"none","verticalAlign":"middle","width":80,"zIndex":32768,"textAlign":"center","backgroundColor":"transparent","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","borderWidth":0,"lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","Pathway"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"id9ee2ea1d":{"id":"id9ee2ea1d","citations":["publicationXrefb7a"],"borderWidth":1,"zIndex":12288,"points":[{"isAttachedTo":"e114e","attachmentDisplay":{"position":[0.5,1],"offset":[0,0]},"x":148.2084690553746,"y":194.03908794788273,"orientation":[0,1]},{"isAttachedTo":"c9fbc","attachmentDisplay":{"position":[0.5,0],"offset":[0,0]},"x":195.4397394136808,"y":328.76221498371376,"orientation":[0,1]}],"color":"#000000","drawAs":"StraightLine","type":["Interaction","Edge"],"kaavioType":"Edge","gpmlElementName":"Interaction","markerEnd":"Arrow","isAttachedTo":["e114e","c9fbc"]},"id8ee9f3be":{"id":"id8ee9f3be","citations":["publicationXrefb8e","publicationXrefe88"],"borderWidth":1,"zIndex":12288,"points":[{"isAttachedTo":"c9fbc","attachmentDisplay":{"position":[1,0.5],"offset":[0,0]},"x":235.4397394136808,"y":338.76221498371376,"orientation":[1,0]},{"x":311.10708469055373,"y":234.5276872964169}],"color":"#000000","drawAs":"StraightLine","type":["Interaction","Edge"],"kaavioType":"Edge","gpmlElementName":"Interaction","markerEnd":"Arrow","isAttachedTo":["c9fbc"]},"id788e2f55":{"id":"id788e2f55","borderWidth":1,"zIndex":12288,"points":[{"x":285.01628664495115,"y":262.214983713355},{"isAttachedTo":"e114e","attachmentDisplay":{"position":[1,0.5],"offset":[0,0]},"x":188.2084690553746,"y":184.03908794788273,"orientation":[-1,0]}],"color":"#000000","drawAs":"StraightLine","type":["Interaction","Edge"],"kaavioType":"Edge","gpmlElementName":"Interaction","markerEnd":"Arrow","isAttachedTo":["e114e"]},"publicationXrefc86":{"id":"publicationXrefc86","dbId":"346","dbConventionalName":"PubMed","standardName":"Treatment of severe hypertension with minoxidil.","source":"Isr J Med Sci","year":1975,"authors":["Traub YM","Redmond DP","Rosenfeld JB","McDonald RH Jr","Shapiro AP"],"type":["PublicationXref"],"gpmlElementName":"PublicationXref","textContent":"1"},"publicationXrefba5":{"id":"publicationXrefba5","dbId":"25","dbConventionalName":"PubMed","standardName":"Pharmacological properties of new neuroleptic compounds.","source":"Arzneimittelforschung","year":1975,"authors":["Coscia L","Causa P","Giuliani E","Nunziata A"],"type":["PublicationXref"],"gpmlElementName":"PublicationXref","textContent":"2"},"publicationXrefcf5":{"id":"publicationXrefcf5","dbId":"326","dbConventionalName":"PubMed","standardName":"Schiff-type dyes in the cytochemical staining of DNA.","source":"Indian J Exp Biol","year":1975,"authors":["Dutt MK"],"type":["PublicationXref"],"gpmlElementName":"PublicationXref","textContent":"3"},"publicationXrefe88":{"id":"publicationXrefe88","dbId":"272","dbConventionalName":"PubMed","standardName":"[Extrabronchial effects of Bronchodilat in patients with asthma and chronic asthmatic bronchitis].","source":"Gruzlica","year":1975,"authors":["Rozniecki J","Grabski W"],"type":["PublicationXref"],"gpmlElementName":"PublicationXref","textContent":"4"},"publicationXrefd54":{"id":"publicationXrefd54","dbId":"363","dbConventionalName":"PubMed","standardName":"Regulation of the hut operons of Salmonella typhimurium and Klebsiella aerogenes by the heterologous hut repressors.","source":"J Bacteriol","year":1975,"authors":["Gerson SL","Magasanik B"],"type":["PublicationXref"],"gpmlElementName":"PublicationXref","textContent":"5"},"publicationXrefb4f":{"id":"publicationXrefb4f","dbId":"264","dbConventionalName":"PubMed","standardName":"5-Carbamoylmethyluridine: a new minor nucleoside of transfer ribonucleic acid.","source":"Biochem Soc Trans","year":1975,"authors":["Dunn DB","Trigg MM"],"type":["PublicationXref"],"gpmlElementName":"PublicationXref","textContent":"6"},"publicationXrefc0d":{"id":"publicationXrefc0d","dbId":"2451","dbConventionalName":"PubMed","standardName":"The morning after: residual EEG effects of triazolam and flurazepam, alone and in combination with alcohol.","source":"Curr Ther Res Clin Exp","year":1976,"authors":["Mendelson WB","Goodwin DW","Hill SY","Reichman JD"],"type":["PublicationXref"],"gpmlElementName":"PublicationXref","textContent":"7"},"publicationXrefb7a":{"id":"publicationXrefb7a","dbId":"2324","dbConventionalName":"PubMed","standardName":"Intramembrane particle aggregation in erythrocyte ghosts. II. The influence of spectrin aggregation.","source":"Biochim Biophys Acta","year":1976,"authors":["Elgsaeter A","Shotton DM","Branton D"],"type":["PublicationXref"],"gpmlElementName":"PublicationXref","textContent":"8"},"publicationXrefee4":{"id":"publicationXrefee4","dbId":"34578","dbConventionalName":"PubMed","standardName":"Rapid screening for urease inhibitors.","source":"Invest Urol","year":1979,"authors":["Hamilton-Miller JM","Gargan RA"],"type":["PublicationXref"],"gpmlElementName":"PublicationXref","textContent":"9"},"publicationXrefe0d":{"id":"publicationXrefe0d","dbId":"6798456","dbConventionalName":"PubMed","standardName":"Stimulus control and drug dependence.","source":"NIDA Res Monogr","year":1981,"authors":["Jasinski DR"],"type":["PublicationXref"],"gpmlElementName":"PublicationXref","textContent":"10"},"publicationXrefb8e":{"id":"publicationXrefb8e","dbId":"15719025","dbConventionalName":"PubMed","standardName":"The role of BH3-only proteins in the immune system.","source":"Nat Rev Immunol","year":2005,"authors":["Strasser A"],"type":["PublicationXref"],"gpmlElementName":"PublicationXref","textContent":"11"}}}
